src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -822,7 +822,14 @@ def make_donor(self, donor):
         }
 
     def make_donors(self, entry):
-        return [self.make_donor(donor) for donor in entry["contents"]["donors"]]
+        donors = entry["contents"]["donors"]
+        if donors:
+            count = donors[0]['donor_count']
+            result = [self.make_donor(donor) for donor in donors]
+            result[0]['donorCount'] = count
+            return result
+        else:
+            return []
```

Comment:
I'm confused why this change made here. I think instead you should be changing the `self.make_donor()` function.

Reply:
I'll leave this to @hannes-ucsc's discretion.

Reply:
Unless I'm mistaken, `len(entry["contents"]["donors"]) < 2` invariantly. If, in the future, we started to group donors by some property (like we do for files which are grouped by format), then there may be more than one donor in an aggregate document. In that case we'd want a count in every donor in the response. My instructions where imprecise in that respect. I think we should just modify make_donor as @jessebrennan suggests, and have the count in every donor in the response.